CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 option(USE_LLD "Use lld (from llvm) linker" OFF)
 option(USE_MOLD "Use mold (A Modern Linker)" OFF)
 option(TI_WITH_BACKTRACE "Use backward-cpp to print out C++ stack trace upon failure" OFF)  # wheel-tag: bt
+option(TI_GENERATE_PDB "Generate Program Database (PDB) files (will make compilation uncacheable)" OFF)
```

Comment:
nit: how about merging this with `DEBUG=1`?

Reply:
No, PDBs are generated by default on debug builds, this is for generating PDBs with release builds.